Chore: Protobuf guide and re-generate (#971)
grafana/grafana-plugin-sdk-go: contribute/developer-guide.md

A prerequisite is to have [protoc](http://google.github.io/proto-lens/installing-protoc.html) installed and available in your path.

> Next, you need to have [protoc-gen-go](https://github.com/protocolbuffers/protobuf-go/tree/master/cmd/protoc-gen-go) and [protoc-gen-grpc-go](https://pkg.go.dev/google.golang.org/grpc/cmd/protoc-gen-go-grpc) installed and available in your path. It's very important that you match the `protoc-gen-go` version specified of `google.golang.org/protobuf` in [go.mod](../go.mod) file, as of this writing it's v1.33.0.

Reviewer: ```suggestion
Next, you need to have [protoc-gen-go](https://github.com/protocolbuffers/protobuf-go/tree/master/cmd/protoc-gen-go) and [protoc-gen-grpc-go](https://pkg.go.dev/google.golang.org/grpc/cmd/protoc-gen-go-grpc) installed and available in your path. It's very important that you match the `protoc-gen-go` version specified of `google.golang.org/protobuf` in [go.mod](../go.mod), which at the time of writing is v1.33.0.
```
